### PR TITLE
Update storage info

### PR DESCRIPTION
**Pull request by szarnyasg**
Update storage information

Related documentation page: https://duckdb.org/docs/internals/storage#storage-version-table

### CODE REVIEW

File: src/storage/storage_info.cpp [hunk 1]
```diff
@@ -22,19 +22,18 @@ struct SerializationVersionInfo {
...
-    {"v0.5.0", 38}, {"v0.5.1", 38}, {"v0.6.0", 39}, {"v0.6.1", 39}, {"v0.7.0", 43},  {"v0.7.1", 43},  {"v0.8.0", 51},
-    {"v0.8.1", 51}, {"v0.9.0", 64}, {"v0.9.1", 64}, {"v0.9.2", 64}, {"v0.10.0", 64}, {"v0.10.1", 64}, {"v0.10.2", 64},
-    {nullptr, 0}};
+    {"v0.0.4", 1},   {"v0.1.0", 1},  {"v0.1.1", 1},  {"v0.1.2", 1},  {"v0.1.3", 1},   {"v0.1.4", 1},   {"v0.1.5", 1},
+    {"v0.1.6", 1},   {"v0.1.7", 1},  {"v0.1.8", 1},  {"v0.1.9", 1},  {"v0.2.0", 1},   {"v0.2.1", 1},   {"v0.2.2", 4},
+    {"v0.2.3", 6},   {"v0.2.4", 11}, {"v0.2.5", 13}, {"v0.2.6", 15}, {"v0.2.7", 17},  {"v0.2.8", 18},  {"v0.2.9", 21},
+    {"v0.3.0", 25},  {"v0.3.1", 27}, {"v0.3.2", 31}, {"v0.3.3", 33}, {"v0.3.4", 33},  {"v0.3.5", 33},  {"v0.4.0", 33},
+    {"v0.5.0", 38},  {"v0.5.1", 38}, {"v0.6.0", 39}, {"v0.6.1", 39}, {"v0.7.0", 43},  {"v0.7.1", 43},  {"v0.8.0", 51},
+    {"v0.8.1", 51},  {"v0.9.0", 64}, {"v0.9.1", 64}, {"v0.9.2", 64}, {"v0.10.0", 64}, {"v0.10.1", 64}, {"v0.10.2", 64},
+    {"v0.10.3", 64}, {"v1.0.0", 64}, {nullptr, 0}};
```
Comment (carlopi) [inline src/storage/storage_info.cpp:31]: Should we add also 1.1.0 here? (given we know it will be on 64) ?

File: src/storage/storage_info.cpp [hunk 2]
```diff
@@ -22,19 +22,18 @@ struct SerializationVersionInfo {
...
 // END OF STORAGE VERSION INFO
 
 // START OF SERIALIZATION VERSION INFO
-static const SerializationVersionInfo serialization_version_info[] = {{"v0.10.0", 1}, {"v0.10.1", 1}, {"v0.10.2", 1},
-                                                                      {"v0.10.3", 2}, {"v1.0.0", 2},  {"v1.1.0", 3},
-                                                                      {"latest", 3},  {nullptr, 0}};
+static const SerializationVersionInfo serialization_version_info[] = {
+    {"v0.10.0", 1}, {"v0.10.1", 1}, {"v0.10.2", 1}, {"latest", 2}, {nullptr, 0}};
```
Comment (carlopi) [inline src/storage/storage_info.cpp:36]: This change I don't get, I think it was better before / giving some DB might already serialized with 3 we can't go back to 2.

**Review by carlopi — COMMENTED** [review]
I think it makes sense to prepare this to 1.1.0, left two questions, not completely sure of the answers.

File: src/storage/storage_info.cpp (hunk 2 above)
Comment (szarnyasg) [reply src/storage/storage_info.cpp]: This was manually edited and overwritten by the generator script which used the JSON. I fixed the JSON now.

**Comment by Mytherin** [comment]
Thanks!

**Pull request merged**